src/lib/viewers/box3d/image360/__tests__/Image360Controls-test.tsx

```diff
@@ -24,11 +22,10 @@ describe('lib/viewers/box3d/image360/Image360Controls', () => {
                 onVrToggle,
             });
 
-            expect(wrapper.find(VrToggleControl).props()).toMatchObject({
-                isVrShown: false,
-                onVrToggle,
-            });
-            expect(wrapper.find(FullscreenToggle).prop('onFullscreenToggle')).toEqual(onFullscreenToggle);
+            await expect(wrapper.queryByTitle('Toggle VR display')).toBe(null);
```

Comment:
Ditto. You do not need wrapper to find elements that were rendered - use `screen` instead. 
Also, I think all the `await` could probably omitted.

Source: [Not using screen](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen)

Reply:
> _The base queries from DOM Testing Library require you to pass a container as the first argument. Most framework-implementations of Testing Library provide a pre-bound version of these queries when you render your components with them which means you do not have to provide a container. In addition, if you just want to query document.body then you can use the [screen](https://testing-library.com/docs/queries/about/#screen) export as demonstrated below (**using screen is recommended**)._

Source: [Using Queries](https://testing-library.com/docs/queries/about/#using-queries)